src/collective/fingerpointing/tests/test_workflow_subscriber.py

```diff
@@ -27,11 +28,18 @@ def setUp(self):
         api.portal.set_registry_record(record, False)
 
     def test_workflow_transitions(self):
-        expected = (
-            ('collective.fingerpointing', 'INFO', u'user=test_user_1_ ip=None action=workflow transition object=<NewsItem at foo> transition=submit'),
-            ('collective.fingerpointing', 'INFO', u'user=test_user_1_ ip=None action=workflow transition object=<NewsItem at foo> transition=publish'),
-            ('collective.fingerpointing', 'INFO', u'user=test_user_1_ ip=None action=workflow transition object=<NewsItem at foo> transition=retract'),
-        )
+        if IS_PLONE_5:
+            expected = (
+                ('collective.fingerpointing', 'INFO', u'user=test_user_1_ ip=None action=workflow transition object=<NewsItem at foo> transition=submit'),  # noqa: C901
```

Comment:
here are the typos again: `E501`.

Reply:
will replace all those... had the wrong number in mind all the time.